litellm/tests/test_bedrock_completion.py

```diff
@@ -70,6 +70,18 @@ def test_completion_bedrock_claude_completion_auth():
         # Add any assertions here to check the response
         print(response)
 
+        # Second call to test cross-region inference
```

Comment:
Can you use Mock calls / Async Mock calls. See `test_sagemaker`

Reply:
sounds good, hoping to get to this at a later point! if anyone else gets to this before i do, would be okay 👍🏻